presto-orc/src/test/java/com/facebook/presto/orc/TestingOrcPredicate.java

```diff
@@ -83,57 +83,57 @@ public static OrcPredicate createOrcPredicate(int columnIndex, Type type, Iterab
     {
         List<Object> expectedValues = newArrayList(values);
         if (BOOLEAN.equals(type)) {
-            return new BooleanOrcPredicate(columnIndex, expectedValues, format == DWRF);
+            return new BooleanOrcPredicate(columnIndex, expectedValues, false);
         }
         if (TINYINT.equals(type) || SMALLINT.equals(type) || INTEGER.equals(type) || BIGINT.equals(type)) {
             return new LongOrcPredicate(true,
                     columnIndex,
                     expectedValues.stream()
                             .map(value -> value == null ? null : ((Number) value).longValue())
                             .collect(toList()),
-                    format == DWRF);
+                    false);
```

Comment:
`createOrcPredicate` is used by `OrcTester` and is used by all block tests. Is there any reason you set the `noFileStats` to false for all the tests? The tests could be on other formats like ORC, are they all without file stats?